articles/security/bulletins/index.md

```diff
@@ -0,0 +1,22 @@
+---
+classes: topic-page
+title: Auth0 Security Bulletins
+description: List of bulletins addressing security vulnerabilities in Auth0 software, with info on how to fix them.
+---
+<div class="topic-page-header">
+  <div data-name="example" class="topic-page-badge"></div>
+  <h1>Auth0 Security Bulletins</h1>
+  <p>
+    List of bulletins addressing security vulnerabilities in Auth0 software.
+  </p>
+</div>
+
+This page contains a list of all the published security vulnerabilities of Auth0 software.
+
+Each bulletin contains a description of the vulnerability, how to identify if you are affected, and what to do to fix it.
+
```

Comment:
@goduncan Do we have more info to add here? Like how to get notifications on future bulletins or updates, or which are our sources for vulnerabilities? Or any info/link to our whitehat program?

Reply:
I don't think we are doing to notify outside of a monthly newsletter @mgonto ?

Reply:
We can add a link to the whitehat Disclosure though: https://auth0.com/whitehat